src/scripts/dotnet-compile-native.bat

```diff
@@ -40,6 +41,7 @@ if /i "%1" == "/logpath" (set __LogFilePath=%2&shift&shift&goto Arg_Loop)
 if /i "%1" == "/codegenpath" (set __CodegenPath=%2&shift&shift&goto Arg_Loop)
 if /i "%1" == "/objgenpath" (set __ObjgenPath=%2&shift&shift&goto Arg_Loop)
 if /i "%1" == "/linklibs" (set __LinkLibs=%2&shift&shift&goto Arg_Loop)
+if /i "%1" == "/reference" (set __AdditionalRefs=%__AdditionalRefs% /r %2&shift&shift&goto Arg_Loop)
```

Comment:
Just a note, any path with spaces would have to be quoted appropriately by the caller.